minimal_redux_poc/src/index.js

```diff
@@ -10,17 +10,30 @@ import './styles/index.scss';
 /**
  * Default Mirador instantiation
  */
-export default function Mirador(config) {
+const Mirador = function(config) {
```

Comment:
I wonder if there is a way we can return the ReactDom ref also here.

I feel like we may want to actually return an Object with more functionality to do things on.

For example, what about something like this?

```javascript
var myMirador = new Mirador(config);
myMirador.registerPlugins([keyboardPlugin, rulerPlugin]);
```
This doesn't necessitate the global Mirador function taking side effects, which could produce unintended consequences? I'm thinking back to our implementations of libraries like i18n and Handlebars (https://github.com/ProjectMirador/mirador/pull/1335).

A page may include multiple instances of Mirador, and those instances may have differing configurations or plugins, right? Would modifying the global `Mirador` object here then prevent that?

Reply:
Yes, we want to avoid any side effects. We also discussed the multiple-instance issue. In practice, plugins, packaging, and the API, initialisation, and configuration concerns are all related to one another, so we haven't addressed that all here. Basically this just shows a way plugins (and an external use of the runtime api) can be bootstrapped with only a script tag (not additional configuration after initialisation)).